pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/index/readers/BitSlicedRangeIndexReader.java

```diff
@@ -94,6 +94,32 @@ public int getNumMatchingDocs(double min, double max) {
     return queryRangeBitmapCardinality(FPOrdering.ordinalOf(min), FPOrdering.ordinalOf(max), 0xFFFFFFFFFFFFFFFFL);
   }
 
+  @Override
+  public int getNumMatchingDocs(int value) {
+    if (value < _min) {
+      return 0;
+    }
+    return queryRangeBitmapCardinality(Math.max(value, _min) - _min, _max - _min);
```

Comment:
Can be simplified, same for long type
```suggestion
    return queryRangeBitmapCardinality(value - _min, _max - _min);
```